<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>outpatient</class>
 <widget class="QWidget" name="outpatient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>681</width>
     <height>451</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Booked</string>
      </property>
      <widget class="QTableView" name="tableView">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>311</width>
         <height>231</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>New booking</string>
      </property>
      <widget class="QWidget" name="gridLayoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>311</width>
         <height>51</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Choose the speciality</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QComboBox" name="specialities">
          <item>
           <property name="text">
            <string>Allergy and Immunology</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Rheumatology</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Gynecology</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Ophthalmology</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Pediatrics</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Urology</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Dermatology</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Pathology</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Cardiology</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Neurology</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="submit_spe">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>submit</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QCalendarWidget" name="calendarWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>312</width>
         <height>190</height>
        </rect>
       </property>
       <property name="dateEditEnabled">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>410</y>
         <width>311</width>
         <height>41</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="submit_booking">
          <property name="text">
           <string>submit</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="reset">
          <property name="text">
           <string>reset</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="submit_date">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>290</y>
         <width>75</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>submit</string>
       </property>
      </widget>
      <widget class="QLabel" name="doctor_label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>310</y>
         <width>159</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Choose the doctor</string>
       </property>
      </widget>
      <widget class="QComboBox" name="doctors">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>330</y>
         <width>311</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
